08b_Interactive_Catalog_Visualization.ipynb

```diff
@@ -242,8 +238,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "assert data[data[\"truth_type\"] == \"star\"].shape[0] == 2164\n",
-    "assert data[data[\"truth_type\"] == \"galaxy\"].shape[0] == 99932\n",
+    "assert data[data[\"truth_type\"] == \"star\"].shape[0] == 331\n",
+    "assert data[data[\"truth_type\"] == \"galaxy\"].shape[0] ==  14472\n",
     "assert data[data[\"truth_type\"] == \"SNe\"].shape[0] == 0"
```

Comment:
I think it got lost in outdated comments, but why are you mentioning supernovae at all?  There aren't any in this dataset and it just seems to promise something this notebook will never deliver.